### DISCUSSION TITLE
Stream Wallpapers not working

### DISCUSSION THREAD

**Discussion by tashi21** [rocksdanister/lively#819] [Q&A]
Hi, the wallpapers I am using from YouTube as YouTube links are showing YouTube banners at the top (the title, "share more", etc.). Often the wallpaper hangs (not the app) and I get this screen:

![Screenshot (8)](https://user-images.githubusercontent.com/79501323/137811808-3731515c-a44a-4b28-abcf-6ae5f8d27bce.png)

Every time I change the wallpapers, I get a notification from my malware detector (Norton) telling me that CefSharp.BrowserSubprocess is requesting access to my camera (which I deny). 

Are my wallpapers not displaying properly because I deny access? Or is it a Windows 11 compatibility issue? Additionally, why is CefSharp.BrowserSubprocess requesting access to my camera in the first place?

Thank you for any help.

**Comment by rocksdanister** [comment]
AV programs have problem with Lively https://github.com/rocksdanister/lively/issues/358

Webcam and mic is blocked by default in CefSharp, can be verified by setting wallpaper:
https://webcamtests.com/

Source code of browser:
https://github.com/rocksdanister/lively-cef/blob/a498e7254f0ed2876dd3cc209c32103068ad9779/src/cefsharptest/Form1.cs#L516

**Reply by tashi21** [reply]
But this doesn't solve my issue of how the wallpapers are being rendered. I don't mind the camera access request
